cmd/paramgen/internal/paramgen.go

```diff
@@ -123,7 +124,7 @@ func parsePackage(path string) (*ast.Package, error) {
 		return nil, fmt.Errorf("no source-code package in directory %s", path)
 	}
 	if len(pkgs) > 1 {
-		return nil, fmt.Errorf("multiple packages in directory %s", path)
+		return nil, fmt.Errorf("multiple packages [%v] in directory %s", maps.Keys(pkgs), path)
```

Comment:
We should remove the brackets, otherwise this will print `[[key1 key2]]`.
```suggestion
		return nil, fmt.Errorf("multiple packages %v in directory %s", maps.Keys(pkgs), path)
```